src/Mono.Android/Android.Runtime/Logger.cs

```diff
@@ -58,18 +61,28 @@ internal static class Logger {
 					System.Console.WriteLine ("[{0}] {1}: {2}", level, appname, line);
 			}
 		}
+
+		[DllImport ("__Internal")]
+		extern static uint monodroid_get_log_categories ();
+
+		static Logger ()
+		{
+			lock (initLock) {
+				logCategories = (LogCategories) monodroid_get_log_categories ();
+			}
+		}
 	}
 
 	internal enum LogLevel {
-		Unknown,
-		Default,
-		Verbose,
-		Debug,
-		Info,
-		Warn,
-		Error,
-		Fatal,
-		Silent
+		Unknown = 0x00,
```

Comment:
Why the "noise"?

For that matter, why *hex* for values which *aren't* flags/bitfields?

Reply:
The "noise" is to keep it explicit since the enum is "shared" with `C++` - it's better to have the values stated clearly IMO. And why hex? I like the way hex numbers look, no other reason :)